improver_tests/wxcode/wxcode/test_WeatherSymbols.py

```diff
@@ -47,186 +47,118 @@
-
-    data_rainv = np.array([0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0, 1.0, 1.0,
-                           0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 1.0,
-                           0.0, 0.0, 0.0, 0.0, 0.0, 0.0,
-                           0.0, 0.0, 0.0]).reshape((3, 1, 3, 3))
-    rainfall_vicinity = (
-        set_up_probability_threshold_cube(
-            data_rainv,
-            'rainfall_rate_in_vicinity',
-            'm s-1',
-            spp__relative_to_threshold='above',
-            forecast_thresholds=np.array([8.33333333e-09,
-                                          2.77777778e-08,
-                                          2.77777778e-07])))
-
-    data_cloud = np.array([0.0, 1.0, 0.0, 0.0, 1.0, 1.0, 0.0, 0.0, 1.0,
-                           0.0, 0.0, 0.0, 0.0, 1.0, 1.0,
-                           0.0, 0.0, 1.0]).reshape((2, 1, 3, 3))
-    cloud = (set_up_probability_threshold_cube(
-        data_cloud,
-        'cloud_area_fraction',
-        '1',
-        spp__relative_to_threshold='above',
-        forecast_thresholds=np.array([0.1875, 0.8125])))
-
-    data_cld_low = np.array([0.0, 0.0, 0.0, 0.0, 0.0, 1.0,
-                             0.0, 0.0, 0.0]).reshape((1, 1, 3, 3))
-    cloud_low = (
-        set_up_probability_threshold_cube(
-            data_cld_low,
-            'low_type_cloud_area_fraction',
-            '1',
-            spp__relative_to_threshold='above',
-            forecast_thresholds=np.array([0.85])))
-
-    data_vis = np.array([0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 0.0,
-                         0.0, 0.0, 1.0, 1.0, 0.0, 0.0,
-                         0.0, 1.0, 0.0]).reshape((2, 1, 3, 3))
-    visibility = (
-        set_up_probability_threshold_cube(
-            data_vis,
-            'visibility_in_air',
-            'm',
-            spp__relative_to_threshold='below',
-            forecast_thresholds=np.array([1000.0, 5000.0])))
-    visibility.attributes['relative_to_threshold'] = 'below'
-
-    data_lightning = np.zeros((1, 1, 3, 3))
-    lightning = (
-        set_up_probability_threshold_cube(
-            data_lightning,
-            'number_of_lightning_flashes_per_unit_area_in_vicinity',
-            'm-2',
-            spp__relative_to_threshold='above',
-            forecast_thresholds=np.array([0.0])))
-
-    cubes = iris.cube.CubeList([snowfall_rate, rainfall_rate,
-                                snowfall_vicinity, rainfall_vicinity,
-                                cloud, cloud_low,
-                                visibility, lightning])
-
-    cubes_no_lightning = iris.cube.CubeList([snowfall_rate, rainfall_rate,
-                                             snowfall_vicinity,
-                                             rainfall_vicinity,
-                                             cloud, cloud_low,
-                                             visibility])
-
-    return cubes, cubes_no_lightning
-
-
-def set_up_wxcubes_global():
```

Comment:
Cubes all set up in one place now and the necessary ones for any given test are simply extracted from the complete list.